### PR TITLE
added a route to get row by key

### PR DESCRIPTION
**Pull request by Henry-Tottle**
Not sure if this will flag the rest of the code that I accidentally did in main

### CODE REVIEW

File: app/routes.php [hunk 1]
```diff
@@ -3,21 +3,12 @@
 
 use App\Controllers\CoursesAPIController;
 use Slim\App;
-use Slim\Views\PhpRenderer;
-use Slim\Interfaces\RouteCollectorProxyInterface as Group;
+
 
 return function (App $app) {
+    //could get rid of this line? not sure what it's doing
```
Comment (joseph-padfield) [inline app/routes.php:9]: I'm pretty sure this is what gets our dependencies...

Comment (Henry-Tottle) [reply app/routes.php:9]: probably,  php storm greys out $container but maybe that's because it doesn't like slim

**Review by joseph-padfield — APPROVED** [review]
All good dude. Not necessary at this stage but have got into the habit of making an abstract controller class for handling the JSON. Could be helpful as the project grows.

File: app/routes.php (hunk 1 above)
Comment (joseph-padfield) [reply app/routes.php:9]: Yeah I think it's a Slim thing. Is that what they call a magic method?

Comment (Henry-Tottle) [reply app/routes.php:9]: it is now 😅

**Pull request merged**